xray/audit/jas/applicabilitymanager.go

```diff
@@ -75,6 +74,10 @@ func newApplicabilityScanManager(xrayScanResults []services.ScanResponse, depend
 	cleanup = func() error {
 		return fileutils.RemoveTempDir(tempDir)
 	}
+	fullPathWorkingDirs, err := utils.GetFullPathsWorkingDirs(workingDirs)
```

Comment:
don't we already do it on frogbot?

Reply:
@sverdlov93, if possible I will take this function in Frogbot.